library/tinkerpop/src/test/java/uk/gov/gchq/gaffer/tinkerpop/GafferPopVertexTest.java

```diff
@@ -241,7 +242,10 @@ void shouldDelegateEdgesToGraph() {
         final GafferPopGraph graph = mock(GafferPopGraph.class);
         final GafferPopVertex vertex = new GafferPopVertex(TestGroups.ENTITY, GafferPopGraph.ID_LABEL, graph);
         final Iterable<Edge> resultEdges = Arrays.asList(((Edge) new GafferPopEdge(GafferPopGraph.ID_LABEL, vertex, vertex, graph)));
-        given(graph.edges(GafferPopGraph.ID_LABEL, new String[]{TestGroups.ENTITY})).willReturn(resultEdges.iterator());
+        given(graph.edgesWithView(GafferPopGraph.ID_LABEL, Direction.IN, new View.Builder()
+                .edges(List.of(TestGroups.ENTITY))
```

Comment:
This doesn't compile. [`Collections.singletonList`](https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#singletonList-T-) should work instead. Or just `Arrays.asList` which is already imported.
```suggestion
                .edges(Collections.singletonList(TestGroups.ENTITY))
```

Reply:
Yep, just spotted this! I had Java 11 installed for some reason